js/app-super-resolution/instructions.md

```diff
@@ -0,0 +1,103 @@
+## Steps of building project from scratch
```

Comment:
The instruction seems only work for Windows system. Is it expected?

Reply:
I believe it should only work for Windows. I am not really familiar with MacOS or Linux systems